tools/depends/target/smctemp/Makefile

```diff
@@ -0,0 +1,30 @@
+include ../../Makefile.include SMCTEMP-VERSION ../../download-files.include
+DEPS = ../../Makefile.include SMCTEMP-VERSION  Makefile ../../download-files.include \
+
+LIBDYLIB=$(PLATFORM)/$(BYPRODUCT)
+
+ifeq ($(CPU), x86_64)
+	TARGET_ARCH = x86_64
+else
+    TARGET_ARCH = arm64
+endif
+
+all: .installed-$(PLATFORM)
+
+$(PLATFORM): $(DEPS) | $(TARBALLS_LOCATION)/$(ARCHIVE).$(HASH_TYPE)
+	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
+	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
+
+$(LIBDYLIB): $(PLATFORM)
+    $(MAKE) -C ARCH=$(TARGET_ARCH) $(PLATFORM) staticlib
```

Comment:
Tabs v spaces leads to bad things in makefiles. This needs to be a TAB (any line in a rule requires tabs, or its not valid. A rule being "something:", outside of the rules you can use spaces). 
-C is used to call make against a specific directory, you cant put the env var between -C and the $(platform)

@wsnipex can chime in, but my preference is to export the var rather than inline it. all that means is adding before the all: target above 
```
export TARGET_ARCH
```

You can add the env var before (i think after is valid as well, but im not a fan of that) if you wish to inline it however. 
```
	ARCH=$(TARGET_ARCH) $(MAKE) -C $(PLATFORM) staticlib
```

Reply:
Yeah the -C and the tab was due to lack of sleeping time :)
Kept the inline variable and will wait for wspinex feedback.

PS: I am not sure if setting env vars work with the way smctemp assignments are set (using := ), see https://github.com/narugit/smctemp/pull/19/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R5